handbook/network-security/network-security.md

```diff
@@ -0,0 +1,225 @@
+# Network Security
+
+SystemLink connects test systems to a central server to aggregate data for monitoring and analysis. SystemLink uses industry standard tools and best practices to ensure the SystemLink server and test systems utilize secure networking techniques.
+
+Refer to [Workspaces and Role-based Access Control](/rbac/rbac) for details secure user interactions with SystemLink resources. Refer to [Single Sign-on with OpenID Connect](/openid-connect/openid-connect/) and [Sign on with LDAP](/ldap/ldap/) for details on securely authenticating with SystemLink.
+
+## Network diagram
+
+<figure>
+  <img src="../../img/network-diagram.png" width="500" />
+  <figcaption>The networked components and encrypted protocols used by SystemLink.</figcaption>
+</figure>
+
+!!! note
+    Refer to [Setting up a SystemLink Server](https://www.ni.com/documentation/en/systemlink/2020r4/setup/setting-up-systemlink-server/) for information regarding ports used by SystemLink. d
+
+- **SystemLink App Server**:
```

Comment:
I feel like spelling out "application" throughout this section might improve readability.

Reply:
This is kind of nebulous in target. But I look at some existing docs and they refer to these as targets (https://www.ni.com/documentation/en/systemlink/latest/setup/setting-up-systemlink-client-windows/). I'm going to stick with that for now, but I honestly don't love the term (I don't really love node either but what can you do).

Reply:
Agreed. I changed all instances.